projects/plugins/jetpack/extensions/plugins/payments/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { useCallback } from '@wordpress/element';
+import { external, Icon } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import analytics from '../../../_inc/client/lib/analytics';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+const PaymentsPostPublish = () => {
+	const trackClick = useCallback(
+		() => analytics.tracks.recordEvent( 'jetpack_editor_payments_post_publish_click' ),
+		[]
+	);
+	return (
+		<PluginPostPublishPanel className="jetpack-payments-post-publish-panel" initialOpen>
+			<p className="post-publish-panel__postpublish-subheader">
+				<strong>{ __( 'Start accepting payments', 'jetpack' ) }</strong>
+			</p>
+			<p>
+				{ __(
+					'Insert the Payments block button or the Donations block form -- no plugin required.',
+					'jetpack'
+				) }
+			</p>
+			<div
+				role="link"
+				className="post-publish-panel__postpublish-buttons"
+				tabIndex={ 0 }
+				onClick={ trackClick }
+				onKeyDown={ trackClick }
+			>
+				<Button isPrimary href="https://wordpress.com/payments-donations/" target="_top">
```

Comment:
As an aside, do we need this to be a `Button`?
Core Gutenberg has an [`ExternalLink`](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/index.js) with a few a11y niceties already built-in.

`ExternalLink` also targets `_blank` instead of `_top`, which is probably appropriate for an external link.

Reply:
> It opens a page marked [private] which is not accessible to the outside world.

It is still a draft, as the contents are still a work in progress.

> As an aside, do we need this to be a Button?
> Core Gutenberg has an [ExternalLink](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/index.js?rgh-link-date=2022-02-25T18%3A04%3A08Z) with a few a11y niceties already built-in.

I had gone with `ExternalLink` at first, but ended up moving to a custom solution in 982227536a77f95dc38de47c475258505d38b656 because of the need to add a Tracks event to that link. As I mentioned in the commit message though, I'm not a big fan of that solution :)

This is the same approach we followed for the Anchor integration:
https://github.com/Automattic/jetpack/blob/d35fe2b3327a5be210c3d916df5bf3368fa742c1/projects/plugins/jetpack/extensions/blocks/anchor-fm/editor.js#L79-L90

Maybe we should move to a new component of ours, that would be core's `ExternalLink` + a Tracks implementation?

Reply:
I haven't tried, but I _think_ `ExternalLink` supports `onClick` (or at least it should propagate all its additional props to the `<a>` inside).

So I wonder if my whole suggestion [below](https://github.com/Automattic/jetpack/pull/23118#discussion_r814977658) could be adapted to use `ExternalLink` instead of `Button`. 🤔

Reply:
> I think ExternalLink supports onClick

You're right, it does seem to work! That would simplify things for sure. I just pushed c2149b69288c369b1f1981dfa9b05d5fc09c95b0 to give this all a try.